internal/tools/aws/route53.go

```diff
@@ -123,7 +124,10 @@ func (a *Client) DeleteCNAME(dnsName string, logger log.FieldLogger) error {
 		return err
 	}
 
-	logger.Debugf("AWS route53 response: %s", prettyRoute53Response(resp))
+	logger.WithFields(map[string]interface{}{
```

Comment:
Just a thought cause I know you've typed `map[string]interace{}{` a few times in this PR and in previous ones.. maybe we should define a type in this package `loggerFields` e.g. `type loggerFields map[string]interface{}` just to save some typing in the future? It'd also look nice. 0/5 on this.

Reply:
Yep, that is actually provided by the package so let me clean that up. I had actually meant to get around to fixing that for all the log lines.